src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,40 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.dao.UsersDataIsNotValidException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_CHARACTER_SIZE_PASSWORD = 6;
+    private static final int MIN_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public boolean register(User user) {
+        if (user.getLogin() == null) {
+            return false;
```

Comment:
implement method according to the task
![image](https://user-images.githubusercontent.com/20326858/219009551-76a4822d-8c2c-4867-b935-74b19e67b36e.png)